includes/core/classes/class-setup.php

```diff
@@ -135,6 +136,31 @@ public function block_category( $block_categories ) {
 		return $block_categories;
 	}
 
+	/**
+	 * Display custom admin notice function
+	 *
+	 * @return void
+	 */
+	public function timezone_check_admin_notice() {
+		$timezone = get_option( 'timezone_string' );
+		if ( $timezone ) {
+			return;
+		}
+		$class   = 'notice notice-error is-dismissible';
+		$string1 = __( 'Please set', 'gatherpress' );
+		$string2 = __( 'your timezone', 'gatherpress' );
+		$string3 = __( 'in order to ensure proper GatherPress settings!', 'gatherpress' );
+
+		printf(
+			'<div class="%1$s"><p>%2$s <a href="%3$s">%4$s</a> %5$s</p></div>',
```

Comment:
These are always tricky... Looks like correct way to handle is with a simple `wp_kses` 
example: https://codex.wordpress.org/I18n_for_WordPress_Developers

on page example:
`<p>
<?php
$url = 'http://example.com';
$link = sprintf( wp_kses( __( 'Check out this link to my <a href="%s">website</a> made with WordPress.', 'my-text-domain' ), array(  'a' => array( 'href' => array() ) ) ), esc_url( $url ) );
echo $link;
?>
</p>`